Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -297,11 +312,15 @@ - (void)authorizationController:(ASAuthorizationController *)controller didCompl
     m_controller = WTFMove(controller);
     m_completionHandler = WTFMove(handler);
     m_delegate = adoptNS([[_WKASDelegate alloc] initWithPage:WTFMove(requestData.page) completionHandler:makeBlockPtr([this](ASAuthorization *auth, NSError *error) mutable {
-        ensureOnMainRunLoop([this, auth = retainPtr(auth)]() {
+        ensureOnMainRunLoop([this, auth = retainPtr(auth), retainPtr(error)]() {
```

Comment:
Not new to this patch but what guarantees that `this` is still alive when the lambda runs asynchronously? Odds are that we want to capture a WeakPtr instead.

Reply:
Since I don't know this code that well, I'm going to let @pascoej comment on this before making changes.